csharp/test/Ice/adapterDeactivation/ServantLocatorI.cs

```diff
@@ -36,13 +36,7 @@ public ServantLocatorI()
                 }
             }
 
-            private static void test(bool b)
-            {
-                if (!b)
-                {
-                    throw new System.Exception();
-                }
-            }
+            private static void test(bool b) => global::Test.TestHelper.test(b);
```

Comment:
Is the global:: really necessary?

Reply:
Yes, because we have `Ice.adapterDeactivation.Test` namespace.